<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tabela_TarefaFuncionario_2</class>
 <widget class="QMainWindow" name="tabela_TarefaFuncionario_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(0, 137, 206);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_User">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>471</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Olá {user.nome}  Hoje é dia {datetime.today()}, {datetime.dayToday} </string>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_Relatorio">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 255, 242);
border-style: outset;
border-width: 2px;
border-radius: 15px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>Relatório</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_Contato">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>300</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 255, 242);
border-style: outset;
border-width: 2px;
border-radius: 15px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>Contato</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_AdicionarTarefa">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 255, 242);
border-style: outset;
border-width: 2px;
border-radius: 15px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>Adicionar Tarefa</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_Cadastro">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>380</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 255, 242);
border-style: outset;
border-width: 2px;
border-radius: 15px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>Cadastro</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tabela_TarefaGerente">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>411</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Colaborador </string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tafera</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Prazo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Prioridade</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="botao_Saida">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>440</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 255, 242);
border-style: outset;
border-width: 2px;
border-radius: 15px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>Sair</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
